contracts/PeeranhaUser.sol

```diff
@@ -162,6 +163,53 @@ contract PeeranhaUser is IPeeranhaUser, Initializable, NativeMetaTransaction, Ac
         UserLib.unfollowCommunity(userContext, user, communityId);
     }
 
+    /**
+     * @dev Link messenger account to Peeranha.
+     *
+     * Requirements:
+     *
+     * - Must be an existing user.  
+     */
+    function linkAccount(address user, CommonLib.MessengerType linkedAccountType, string memory handle) public {
+        checkHasRole(_msgSender(), UserLib.ActionRole.Bot, 0);
+        checkUser(user);
+        // linkedAccounts[user][linkedAccountType] = UserLib.LinkedAccount(userId, false);
+        UserLib.LinkedAccount storage account = linkedAccounts[linkedAccountType][CommonLib.stringToBytes32(handle)];
```

Comment:
This logic has to be in `UserLib`. We are trying to keep the contract code thin.